services/storage/tests/unit/test_rest.py

```diff
@@ -207,9 +207,9 @@ async def test_upload_link(client, dsm_mockup_db):
         assert data
 
 
+@pytest.mark.skipif(not has_datcore_tokens(), reason="no datcore tokens")
```

Comment:
ahh shade! A lot of these changes are in  my PR. I should have merged it!

Reply:
I bet.